modules/appeals_api/config/flipper/enabled_features.yml

```diff
@@ -0,0 +1,21 @@
+common:
+  - decision_review_hlr_email
+  - decision_review_nod_email
+  - decision_review_sc_email
+  - decision_review_icn_updater_enabled
+development:
+staging:
+  - decision_review_weekly_error_report_enabled
+  - decision_review_daily_stuck_records_report_enabled
+  - decision_review_monthly_stats_report_enabled
+sandbox:
+production:
+  - decision_review_hlr_pii_expunge_enabled
+  - decision_review_nod_pii_expunge_enabled
+  - decision_review_sc_pii_expunge_enabled
+  - decision_review_hlr_status_updater_enabled
+  - decision_review_nod_status_updater_enabled
+  - decision_review_sc_status_updater_enabled
+  - decision_review_weekly_error_report_enabled
+  - decision_review_daily_stuck_records_report_enabled
+  - decision_review_monthly_stats_report_enabled
```

Comment:
@cilestin I saw your recent PR where you added a new feature flag, `decision_review_daily_error_report_enabled`. Can you share which environments you're planning to enable that feature in, and I'll add it to this config? Thanks!

Reply:
staging & prod

Reply:
Thanks! This is updated.